tests/root_io/read_datasource.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+"""Small test case for checking DataSource based creating RDataFrames is accessible from python"""
+
+import ROOT
+from podio.data_source import CreateDataFrame
+
+if ROOT.gSystem.Load("libTestDataModelDict") < 0:  # noqa: E402
+    raise RuntimeError("Could not load TestDataModel dictionary")
+
+input_file = "example_frame.root"  # pylint: disable-msg=C0103
```

Comment:
What is `C0103`? Can we avoid the pylint directive if we simply do this?
```python
rdf = CreateDataFrame("example_frame.root")
```

Reply:
invalid name for a constant. yeah that will work